src/users/src/users-service/routes.py

```diff
@@ -0,0 +1,79 @@
+from fastapi import APIRouter, HTTPException
+from repository import init, get_all_users, get_user_by_id, get_user_by_username, get_user_by_identity_id, get_unclaimed_users, get_random_user, claim_user, create_user, update_user, verify_and_update_phone
+from pinpoint import send_pinpoint_message
+
+router = APIRouter()
+
+@router.get("/")
+def index():
+    return {"message": "Welcome to the User Service"}
+
+@router.post('/users/init')
+def initialise_users():
+    users_loaded = init()
+    return {
+        "users": users_loaded
+    }
+
+@router.get("/users/all")
+def get_users():
+    users = get_all_users()
```

Comment:
We need to think abut security here.  Obviously just a demo, but we should perform authorization checks for these operations

Reply:
I guess, under what conditions? I'll investigate and see where this api call is used otherwise we can discuss if this should even be available publicly